test/test_transforms.py

```diff
@@ -200,18 +200,20 @@ class TestToTensor:
     def test_to_tensor(self, channels):
         height, width = 4, 4
         trans = transforms.ToTensor()
+        np_rng = np.random.RandomState(0)
```

Comment:
I think we can remove this one,  as you're already declaring another RandomState below

Reply:
my bad yeah i missed that one